pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -219,6 +223,19 @@ type CommonPrometheusFields struct {
 	// Number of seconds to wait until a scrape request times out.
 	ScrapeTimeout Duration `json:"scrapeTimeout,omitempty"`
 
+	// The protocols to negotiate during a scrape. It tells clients what
+	// protocol are accepted by Prometheus and with what weight (most wanted is first).
+	// Supported values (case sensitive): PrometheusProto, OpenMetricsText0.0.1,
+	// OpenMetricsText1.0.0, PrometheusText0.0.4.
+	//
+	// If unset, Prometheus uses its default value: ["OpenMetricsText1.0.0","OpenMetricsText0.0.1","PrometheusText0.0.4"]
```

Comment:
(nit) the default value might change, it's better to stay vague.
```suggestion
	// If unset, Prometheus uses its default value.
```

Reply:
Thank you for your opinion. Done